tools/scylla-nodetool.cc

```diff
@@ -1734,6 +1730,13 @@ std::vector<char*> massage_argv(int argc, char** argv) {
         }
 
         std::string arg = argv[i];
+
+        // Java JVM options, they look like -Dkey=value, or just -Dkey
+        // We leave -Dcom.scylladb.apiPort, it will be substituted to --rest-api-port
+        if (argv[i][1] == 'D' && !arg.starts_with("-Dcom.scylladb.apiPort=")) {
+            continue;
+        }
```

Comment:
this is equivalent to
```c++
if (!arg.starts_with("-Dcom.scylladb.apiPort=")) {
    continue;
}
```

i think.

Reply:
No:
* If arg.startswith("-Dcom.scylladb.apiPort="), I want to fall-though this if and continue executing the loop body
* Otherwise, if arg is some other `-D` JVM switch, I want to execute `continue` (drop the arg).

Reply:
ahh, i see. was confused with `argv[i][1]` and `arg`. might want replace the first one with `arg` though. could be a follow-up cleanup.